tests/checkpoint/test_checkpoint.py

```diff
@@ -2316,19 +2228,6 @@ def fake_cloud_context_with_slack(_fake_cloud_context_setup, monkeypatch):
     yield context, slack_counter
 
 
-@pytest.mark.cloud
-def test_use_validation_url_from_cloud(fake_cloud_context_basic):
```

Comment:
Even if we aren't using this action, we are still saving the data to cloud. This test seems to be asserting we can construct the URL we save to correctly. Do we need a test to replace this?

Reply:
I think we have decent coverage over this in our new V1 checkpoint tests but I'll double check and write tests as needed.